src/nsls2api/services/beamline_service.py

```diff
@@ -120,50 +136,61 @@ async def add_detector(
     return new_detector
 
 
-async def add_service(
+async def delete_detector(
```

Comment:
Just wondering why all the args for this function ?  Doesn't the beamline + detector names specify which detector completely ?

Reply:
I did this to make it match the `add_detector` method and let someone send a full "detector object" if they want to, but I guess that only needs to be done at the endpoint and can be stripped down (name and dir_name are the only things I would expect we enforce to be unique)

Reply:
I've cut this down to just use the name, as we now guarantee it to be unique.